### DISCUSSION TITLE
Does the Downloader Ignore Muted Users?

### DISCUSSION THREAD

**Discussion by ghost** [nianeyna/ao3downloader#82] [Q&A] [answered]
Hello,

I was wondering if the downloader ignores Muted users? I have several, but it seems that unfortunately some keep showing up when I update a series. Does the most recent version of the downloader ignore these users?

Also, I found instruction on how to block a tag via CCS  by a user named najo, would the program recognize a skin if it was set to be on when logged in?

Thank you for your help!

**Comment by nianeyna** [comment, marked as answer]
Very interesting questions - I hadn't thought of this. I would have to experiment a bit to get an answer. I do think that it is desirable that it *would* ignore muted users, though, so I am going to convert this discussion into an issue to work on in the future.
